server/settings.py

```diff
@@ -69,7 +69,7 @@ def env(variable, fallback_value=None):
 
 INSTALLED_APPS = ["planning"]
 
-MODULES.extend(["planning", "planning.content_api.events", "planning.content_api.planning"])
+MODULES.extend(["planning"])
```

Comment:
Is this settings.py file really used anywhere? I couldn't find a reference to it the other day.